azurerm/resource_arm_public_ip.go

```diff
@@ -236,6 +234,10 @@ func resourceArmPublicIpRead(d *schema.ResourceData, meta interface{}) error {
 		} else {
 			d.Set("ip_address", "")
 		}
+
+		if idleTimeout := props.IdleTimeoutInMinutes; idleTimeout != nil {
+			d.Set("idle_timeout_in_minutes", idleTimeout)
```

Comment:
d.Set will handle this value being nil - so we should be able to omit the if statement here?